Added animation for the menu item line of the "header" block when hov…

petrovsky747 opened the pull request:
…ering over a menu item.

Added semantics to the existing layout.
Мade adaptive layout of the "destinations" and "sign up" blocks.

petrovsky747 committed:
Added animation for the menu item line of the "header" block when hovering over a menu item.
Added semantics to the existing layout.
Мade adaptive layout of the "destinations" and "sign up" blocks.

5tronciy commented on index.html:
```diff
@@ -265,8 +279,95 @@ <h2 class="title homes-title">Homes guests loves</h2>
+      <button class="button destinations-button regions">Regions</button>
+      <button class="button destinations-button cities">Cities</button>
+      <button class="button destinations-button places">Places</button>
+    </form>
```
There is nothing to send and it no sense to wrap buttons into a form tag. It is not a form.

5tronciy commented on index.html:
```diff
@@ -265,8 +279,95 @@ <h2 class="title homes-title">Homes guests loves</h2>
+    </div>
+  </div>
+</section>
 <section class="footer">
```
No need to wrap the footer in a section tag if it is wrapped into a footer tag.

5tronciy commented on index.html:
```diff
@@ -89,10 +89,24 @@
+    <path d="M40.7324 25.4806C35.6179 25.4806 31.4648 29.6336 31.4648 34.7482C31.4648 39.8628 35.6179 44.0158 40.7324 44.0158C45.847 44.0158 50 39.8628 50 34.7482C50 29.6336 45.8572 25.4806 40.7324 25.4806ZM46.1539 36.0984H42.0827V40.1696H39.3822V36.0984H35.311V33.3979H39.3822V29.3267H42.0827V33.3979H46.1539V36.0984Z" fill="#3077C6"/>
+  </symbol>
 </svg>
 <section class="topsection">
```
No need to wrap the header in a section tag if it is consist of a header element only.

5tronciy reviewed (changes requested):
It looks good in the browser with different resolutions.
Look throw examples in [these docs](https://html.spec.whatwg.org/multipage/sections.html) to get clear use of semantic elements.

petrovsky747 committed:
Fixed semantics to the existing layout.
Buttons of the "destination" section moved out of the form into the "destinations-tabs" block.
Fixed padding based on perfect pixel.

5tronciy reviewed (approved):
Looks good!
